### PR TITLE
[DX] make doctrine mappings compiler pass exception message more understandable

### PR DESCRIPTION
**Pull request by dbu**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Bridge/Doctrine/DependencyInjection/CompilerPass/RegisterMappingsPass.php [hunk 1]
```diff
@@ -231,7 +231,7 @@ private function getManagerName(ContainerBuilder $container)
             }
         }
 
-        throw new ParameterNotFoundException('None of the managerParameters resulted in a valid name');
+        throw new ParameterNotFoundException('Could not determine the doctrine manager. Either Doctrine is not configured or a bundle is misconfigured.');
```
Comment (javiereguiluz) [inline src/Symfony/Bridge/Doctrine/DependencyInjection/CompilerPass/RegisterMappingsPass.php]: `doctrine manager` -> `Doctrine manager`

**Comment by stof** [comment]
:+1: 

however, 2.6 is not maintained anymore (except for security issues) so this should be merged into 2.7

**Comment by javiereguiluz** [comment]
A big :+1: to any commit that improves error messages! Thanks @dbu.

**Comment by aitboudad** [comment]
:+1:

**Comment by dbu** [comment]
recreated with correct spelling for 2.7: #15501

**Pull request closed without merging**